transactions/stakingProxy/sh_get_node_info.cdc

```diff
@@ -0,0 +1,15 @@
+import StakingProxy from 0x179b6b1cb6755e31
+
+pub fun main(account: Address): StakingProxy.NodeInfo {
+
+    let nodeAccount = getAccount(account)
+
+    let proxyCapability = nodeAccount.getCapability
+        <&StakingProxy.NodeStakerProxyHolder{StakingProxy.NodeStakerProxyHolderPublic}>
+        (StakingProxy.NodeOperatorCapabilityPublicPath)!
+
+    let proxyRef = proxyCapability.borrow()
+        ?? panic("Could not borrow public reference to staking proxy")
+
+    return proxyRef.getNodeInfo()!
```

Comment:
This actually needs to take the `nodeID` as an argument. Probably best to change the function to:

```
pub fun main(account: Address, nodeID: String): StakingProxy.NodeInfo {

Reply:
Good catch! Fixed in https://github.com/onflow/flow-core-contracts/pull/29

Reply:
oops. Yeah I didn't test this one